src/OpenAPM.ts

```diff
@@ -304,6 +323,32 @@ export class OpenAPM extends LevitateEvents {
    */
   public REDMiddleware = this._REDMiddleware;
 
+  public getMetrics = async (): Promise<string> => {
```

Comment:
@chtushar Do we have test cases for covering multiple scenarios in this function?

1. Prisma enabled
2. Prisma not present
3. Simple cache present
4. Simple cache not present

Reply:
No we don't have test cases for this. I'll add them.